cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/Location.java

```diff
@@ -486,6 +488,25 @@ public void setOfficeId(String officeId) {
         this.officeId = officeId;
     }
 
+    public Location active(boolean active) {
+        this.active = active;
+        return this;
+    }
+
+    /**
+     * Get officeId
+     *
```

Comment:
is it me or is something missing here, like the getOfficeId function call itself?

Reply:
whoops, that's a typo

Reply:
resolved